{"name": "Проверка ошибки при получении заказов пользователя без авторизации", "status": "passed", "steps": [{"name": "Получаем заказы конкретного пользователя", "status": "passed", "parameters": [{"name": "access_token", "value": "None"}], "start": 1714325546977, "stop": 1714325547270}], "start": 1714325546977, "stop": 1714325547271, "uuid": "11f9834e-2aa1-4d7a-b160-8b687ac14ab9", "historyId": "c1954f6f4eb6058369536e2022675de9", "testCaseId": "c1954f6f4eb6058369536e2022675de9", "fullName": "tests.test_get_user_ordesr.TestGetUserOrders#test_get_user_orders_without_authorization", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_user_ordesr"}, {"name": "subSuite", "value": "TestGetUserOrders"}, {"name": "host", "value": "WIN-VD1QDI3F4U5"}, {"name": "thread", "value": "1424-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_user_ordesr"}]}